DevCycle.SDK.Server.Local/Api/DVCLocalClient.cs

```diff
@@ -121,7 +121,10 @@ public Variable<T> Variable<T>(User user, string key, T defaultValue)
             {
                 try
                 {
-                    existingVariable = config.Variables.Get<T>(key);
+                    existingVariable = Common.Model.Local.Variable<T>.InitializeFromVariableDictionary(
```

Comment:
you shouldn't change this, there's logic in that Get method that we want here, particular to trigger the cast exception below